### ISSUE TITLE
Newsletter country and language form fields are missing descriptive labels

### ISSUE THREAD

**Issue by alexgibson** [mozilla/bedrock#11829]
## Description

Our newsletter form fields should all have descriptive labels associated with them.

### Steps to reproduce

1. Open https://www.mozilla.org/en-US/ and scroll down to the newsletter section.
2. Focus the email field to expand the form.

### Expected result

The country and language fields should have descriptive labels, like shown in Protocol:

![image](https://user-images.githubusercontent.com/400117/176394795-99249fd2-e57d-47e6-9136-4c1bf519526f.png)

### Actual result

The fields are missing descriptive labels:

![image](https://user-images.githubusercontent.com/400117/176394958-783d9b42-c3cc-4349-8141-ccfab5dbbe05.png)

### Environment

N/A
